### PR TITLE
Viewers audit

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
It's depth first then source object, so I'm confused what's really going on under the hood with this proc.
`viewers(Depth=world.view,Center=usr)`
Many weird usages makes me think it just uses defaults when params of the wrong type are passed in

## Why It's Good For The Game
Bug fixes

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
`Both arguments are optional and may be passed in any order.`

**Comment by jlsnow301** [comment]
> `Both arguments are optional and may be passed in any order.`

I hate it

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@
 			dir_shots.Trigger(target = target)
 
 /mob/living/simple_animal/hostile/megafauna/colossus/proc/telegraph()
-	for(var/mob/viewer as anything in viewers(10, src))
+	for(var/mob/viewer as anything in viewers(DEFAULT_SIGHT_DISTANCE, src))
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm:128]: It's not much but on lavaland it's probably important for people this 3 tile distance away to know that colossus is about to attack

Comment (jlsnow301) [reply code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm:128]: i will test this to see if its actually getting viewers at 10. curious

**Comment by ZephyrTFA** [comment]
it's so over

**Pull request closed without merging**